deployment/helm/templates/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+  #namespace: {{ .Values.settings.env }}
+  labels:
+    app.kubernetes.io/name: {{ .Values.settings.name }}
+    app.kubernetes.io/part-of: {{ .Values.settings.name }}
+    helm.sh/chart: {{ include "chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+  allowed-hosts: {{ .Values.security.allowedHosts | quote }}
+  module: {{ .Values.settings.module }}
+  cache-default: {{ .Values.settings.name }}-cache:{{ .Values.redis.port }}/{{ .Values.redis.database }}
+  cache-axes: {{ .Values.settings.name }}-cache:{{ .Values.redis.port }}/{{ .Values.redis.database }}
+  debug: {{ .Values.settings.debug | quote }}
```

Comment:
this should be left out - we want to make it as hard as possible to run insecure deployments